core/src/blocktree_processor.rs

```diff
@@ -161,6 +160,9 @@ fn process_entries_with_callback(
         }
     }
     execute_batches(bank, &batches, entry_callback)?;
+    for hash in tick_hashes {
+        bank.register_tick(&hash);
```

Comment:
is this guaranteed to be safe? There's no code that can execute that varies its behavior based on tick height?

Reply:
@rob-solana said so.. we only use the hash that is the last hash in the slot

Reply:
I don't think we expose tick_height to programs.

Reply:
we don't expose ticks to programs